alws/routers/errata.py

```diff
@@ -36,26 +38,37 @@
 
 @router.post("/new/", response_model=errata_schema.CreateErrataResponse)
 async def create_new_errata_record(
-    errata: errata_schema.BaseErrataRecord,
-    db: AsyncSession = Depends(AsyncSessionDependency(key=get_async_db_key())),
+    errata: errata_schema.BaseErrataRecord
 ):
-    record = await errata_crud.create_new_errata_record(
-        db,
-        errata,
-    )
-    return {"ok": bool(record)}
+    try:
+        errata_data = errata.model_dump()
+
+        # Convert date fields to strings
+        errata_data["issued_date"] = errata_data["issued_date"].isoformat()
+        errata_data["updated_date"] = errata_data["updated_date"].isoformat()
+
+        create_new_errata.send(errata)
+
+        return {"ok": True}
+    except ValueError as e:
+        return {"ok": False, "error": str(e)}
```

Comment:
`errata_schema.CreateErrataResponse` doesn't have `error` field in its definition.
Could you please check if it works as it is?